openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -293,21 +293,19 @@ def get(self, request):
     def list(self, request):
         """
         GET /api/user/v1/accounts?username={username1,username2}
-        GET /api/user/v1/accounts?email={user_email}
+        GET /api/user/v1/accounts?email={user_email1,user_email2}
```

Comment:
same as above, Shouldn't this be `GET /api/user/v1/accounts?email={user_email1},{user_email2}`